src/benchmark.rs

```diff
@@ -0,0 +1,46 @@
+mod structures;
+mod gtfs_structures;
+mod algo;
+extern crate chrono;
+extern crate cpuprofiler;
+extern crate itertools;
+#[macro_use]
+extern crate serde_derive;
+use chrono::prelude::*;
+use cpuprofiler::PROFILER;
+use itertools::Itertools;
+
+fn main() {
+    let gtfs = gtfs_structures::Gtfs::new("./test_data/idf/").unwrap();
```

Comment:
you may even use just '.', much more flexible, just change the current directory